examples/torch/requirements.txt

```diff
@@ -3,8 +3,7 @@ pillow>=8.0.1
 tensorboard>=2.1
 matplotlib>=3.3.3
 defusedxml>=0.7.0rc1
-mlflow~=2.9
-returns>0.14
+tabulate~=0.9.0
```

Comment:
`returns` is not needed anymore since it was for `SafeMLFlow` only 
`tabulate` is needed for optional dumping of debug information in the AutoQ algorithm